davinci-dome/src/App.tsx

```diff
@@ -49,9 +49,11 @@ function App(): JSX.Element {
         const radians = Math.abs(degreesToRadians(degrees))
         const result = daVinci(new Scaffold(frequency, radius, chiralityFromSpec(renderSpec)), radians, true)
         setDaVinciResult(daVinciToDome(result, renderSpec.planeHeight))
-        console.log("RENDER_SPEC", renderSpec.radius)
         setVersion(v => v + 1)
     }, [renderSpec])
+    const circleRadius = () => 1.5 * Math.sqrt(daVinciResult.joints.filter(j => j.position !== JointPosition.Below).reduce((r, {point}) => (
```

Comment:
dynamically calculate the circle radius so adjusts when the plane is raised

Reply:
nice!!